pandas/tests/series/test_operators.py

```diff
@@ -1216,11 +1216,11 @@ def test_neg(self):
     def test_invert(self):
         assert_series_equal(-(self.series < 0), ~(self.series < 0))
 
-    def test_operators(self):
+    def test_operators(self, absfunc=np.abs):
```

Comment:
is this change related to the PR?

Reply:
This was leftover from the previous method of testing, so I will revert it back.